lib/spack/spack/cmd/tutorial.py

```diff
@@ -0,0 +1,84 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from __future__ import print_function
+
+import os.path
+import shutil
+
+import llnl.util.tty as tty
+from llnl.util.filesystem import working_dir
+
+import spack.cmd.common.arguments as arguments
+import spack.config
+import spack.paths
+import spack.util.gpg
+from spack.util.executable import which
+from spack.util.spack_yaml import syaml_dict
+
+
+description = "set up spack for our tutorial (WARNING: modifies config!)"
+section = "config"
+level = "long"
+
+
+# tutorial configuration parameters
+tutorial_branch = "releases/v0.15"
+tutorial_mirror = "s3://spack-tutorial-container/mirror/"
```

Comment:
I tried this locally with Docker by running the basics section of the tutorial. It works, but retrieving binary packages was slow. Not saying we need to do it right now, but it would be good in the future if this commands understands the context and sets the mirror to `/mirror` when we are running from our Docker images.